### ISSUE TITLE
Add GitHub repository link to user interface

### ISSUE THREAD

**Issue by mfisher87** [nsidc/usaon-benefit-tool#227]
We have access to this icon in our project already: https://icons.getbootstrap.com/icons/github/

**Comment by Sherwin-14** [comment]
Can I take this issue up ?

**Comment by mfisher87** [comment]
@Sherwin-14 that would be great! Please let me know if there's anything I can do to support you. I'm thinking the repo URL should be in the `constants` subpackage, but other than that I don't have any strong opinions about this work :)

**Comment by Sherwin-14** [comment]
Hey! I've been trying to get the latest commit from the master branch to work using the instructions provided on your documentation, I am considerably quite new to Docker so it's a bit hard for me, I am providing you with the logs ttps://paste.rs/SE7Ia

**Comment by mfisher87** [comment]
Oh no! Just to double-check, are you working from `4ab3f6b` (`main` branch)?

I updated the docs last night as well. Are you using this page? https://usaon-benefit-tool.readthedocs.io/en/latest/how-to/development.html#

Thanks for sharing the logs. I did some googling and I can't find any mention of `libmamba Subprocess call failed: Invalid argument`. I attempted a fresh build just now from the latest commit on `main`, and it succeeded for me.

Can you please share some system info? I'm using:

* Ubuntu 22
* `docker --version`:  24.0.5, build ced0996
* `docker compose version`:  v2.20.2

Do you also have at least 2GB of free disk space?

**Comment by mfisher87** [comment]
I just pushed another commit on `main` (`ea32471`) to upgrade the base micromamba image we're building from. Maybe that will help?

Since I wasn't able to reproduce, I'm just guessing based on the error coming from `libmamba`.

**Comment by Sherwin-14** [comment]
Hey there! Sure I can share my system info, I also tried this on the latest Fedora release with no success.

- Arch Linux
- Docker Version: 24.0.7
- Docker Compose: 2.23.3

**Comment by mfisher87** [comment]
>It is currently late so is it alright if I will not be able to check the branch until tomorrow 😓

No problem at all, this feature is not urgent. Please don't feel pressured :) :heart:

**Comment by mfisher87** [comment]
Are you on an ARM machine perhaps?

**Comment by mfisher87** [comment]
I just reproduced your exact symptom by creating a fresh Fedora VM in DigitalOcean. I'm not really understanding how the pip install step could fail on one distro but not another. The builds are succeeding in GitHub Actions (Ubuntu). There's something weird going on here. If I remove the pip dependencies, the build succeeds. 

If you'd like to get going faster, can you comment the `build: "."` line from the `compose.dev.yml` file, so a pre-build image from DockerHub is downloaded?

**Comment by mfisher87** [comment]
I tried (separately) disabling selinux, and changing the version of the base image to 1.0.0, both reproduced the exact same symptom on the Fedora droplet. I couldn't think of anything else. I was able to get a successful build on a Debian droplet. I'm really confused about this, and I really don't know much about Fedora. I'm sorry I don't have other ideas at the moment. One solution, which takes a lot more work, would be to get our `pip` dependencies on to `conda-forge`, since earlier I confirmed removing pip dependencies makes the build succeed.

>If you'd like to get going faster, can you comment the `build: "."` line from the `compose.dev.yml` file, so a pre-build image from DockerHub is downloaded?

This is still my best suggestion :frowning_face:
